.github/workflows/yarn.yaml

```
@@ -0,0 +1,13 @@
+name: Check with yarn
+on: [push, pull_request]
+jobs:
+  build:
+    name: Install and test with yarn
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: "18"
```

```suggestion
          node-version: 16.16.0
```

This matches Cypress' node-version.